custom/ewsghana/templates/ewsghana/base_template.html

```diff
@@ -11,7 +11,7 @@
     <script src="{% static 'ewsghana/js/lib/canvg/StackBlur.min.js' %}"></script>
 {% endblock %}
 
-{% block reports-js-inline %}
+{% block js-inline %}
```

Comment:
yes, think there was a bad merge because of https://github.com/dimagi/commcare-hq/pull/16885. fixed, thank you!